src/test/resources/GenerateWithWorkspaceTest/generate java with gen file override/build.gradle

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) 2022 Andrew Parmet
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+plugins {
+  id 'java'
+  id 'com.parmet.buf'
+}
+
+repositories {
+  mavenCentral()
+}
+
+buf {
+  generate {
+    genFileLocation = project.file("subdir/buf.gen.yaml")
+  }
+}
+
+compileJava.dependsOn 'bufGenerate'
+
+sourceSets.main.java {
+  srcDir 'build/bufbuild/generated/java'
+}
+
+dependencies {
+  buf(fileTree('subdir') { include '*.yaml' })
```

Comment:
Is this buf config dependency needed? If possible tests should not override the default config unless they are specifically testing an override.

Reply:
This isn't needed, I'll remove it